src/bare_etiss_processor/run_helper.sh.in

```diff
@@ -30,7 +30,8 @@ TARGETSW_PATH_ELF=$1
 TARGET_ELFFILE=$(basename -- "$TARGETSW_PATH_ELF")
 TARGET_ELFDIR=$(dirname -- "$TARGETSW_PATH_ELF")
 
-VP_EXE="${ETISS_DIR}/examples/bare_etiss_processor/build/main"
+VP_EXE="${ETISS_DIR}bin/bare_etiss_processor"
```

Comment:
small style improvement: define paths without trailing slash (above in line 5), then this becomes more readable

Reply:
agreed